src/allmydata/util/pollmixin.py

```diff
@@ -1,4 +1,17 @@
+"""
+Polling utility that returns Deferred.
+
+Ported to Python 3.
+"""
+
 from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import unicode_literals
+
+from future.utils import PY2
+if PY2:
+    from builtins import filter, map, zip, ascii, chr, hex, input, next, oct, open, pow, round, super, bytes, dict, int, list, object, range, str, max, min  # noqa: F401
```

Comment:
For new code we seem to be converging around formatting imports like:
```
from X import (
   quux,
   foo,
)
```
...etcetera. However, I don't think we've written that down anywhere (and maybe for builtins it's worth special-casing anyway?)

Reply:
1. Once we finish switching to Python 3 this code can all be deleted.
2. Making it mulitple lines means you need to have the `# noqa` on each line, which makes it even worse.

I'd say just go for this and then delete it all once port is finished.